Pipfile

```diff
@@ -23,3 +23,6 @@ jupyter = "*"
 pandas = "*"
 Pillow = "*"
 jsonpickle = "*"
+imageio = "*"
+matplotlib = "*"
+tensorflow = "*"
```

Comment:
@JWeb56 For whatever reason, trying to `pipenv lock` with `tensorflow = "*"` as a dependency causes an error!